ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -89,9 +90,29 @@ const INITIAL_STATE: State = {
   pan: none,
   expirationDate: none,
   securityCode: none,
-  holder: none
+  holder: none,
+  isValidHolder: undefined,
+  isValidPan: undefined,
+  isValidExpirationDate: undefined,
+  isValidSecurityCode: undefined
 };
 
+function isPanValid(pan: string): boolean {
```

Comment:
this function is just an alias of `CreditCardPan.is`, no need for it, just call `CreditCardPan.is`

Reply:
I committed a version of the code which avoids the usage of these functions by inserting the type control on the same line used to pass the prop isValid (true if the input content satisfies the guard check). I maintained a private function to check it only for the expiration date being a more complex logic. Anyway, this approach seems to be discouraged, having the following error on the render method:
"Refactor this method to reduce its Cognitive Complexity from N to 15"  
(N from 16 to 18)